src/routes/api/roles.js

```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express';
+import Users from '../../controllers/users';
+import validateSetRole from '../../middlewares/userRole';
+import { permit } from '../../middlewares/permissions';
+import isLoggedIn from '../../middlewares/login';
+
+const router = Router();
+
+router.patch('/users/:email/role', [isLoggedIn, validateSetRole, permit], Users.changeRole);
```

Comment:
this api should be in the user routes folder because its under `users`.

Reply:
passing email as a param is not a good practice instead use `userId`

Reply:
Ok. Adjusted